extension/src/dcp/types.ts

```diff
@@ -16,6 +16,7 @@ type LaunchConfigurationMode = "Debug" | "NoDebug";
 export interface ExecutableLaunchConfiguration {
     type: string;
     mode?: LaunchConfigurationMode | undefined;
+    debugger_properties?: Record<string, any>;
```

Comment:
Is there a reason why we limit ourselves to this being a property bag, as opposed to full object?

For comparison, VS Code allows object nesting in their launch configurations, e.g. https://code.visualstudio.com/docs/debugtest/debugging-configuration#_platformspecific-properties